plugins/GSdx/GSDevice11.cpp

```diff
@@ -189,9 +189,19 @@ bool GSDevice11::Create(const std::shared_ptr<GSWnd> &wnd)
 	theApp.LoadResource(IDR_CONVERT_FX, shader);
 	CompileShader(shader.data(), shader.size(), "convert.fx", nullptr, "vs_main", nullptr, &m_convert.vs, il_convert, countof(il_convert), &m_convert.il);
 
+	std::string convert_mstr[1];
+
+	convert_mstr[0] = format("%d", m_upscale_multiplier);
```

Comment:
Maybe use `std::to_string` instead?

Reply:
It's what's done on the other macros and I'd rather keep it the same.

Reply:
Personally I agree with Turtleli, just because someone used an inefficient way to do something before doesn't mean we still have to follow that inefficient approach in the name of consistency.